### PR TITLE
Add Immersive Reader iOS SDK

### PR DESCRIPTION
**Pull request by MeganRoach**
Add the Immersive Reader iOS SDK CocoaPod project and all files.
Add a Swift sample for the Immersive Reader iOS SDK.

### CODE REVIEW

File: iOS/samples/.gitignore [hunk 1]
```diff
@@ -0,0 +1,79 @@
+# Created by https://www.gitignore.io/api/swift,xcode,cocoapods
```
Comment (rwallerms) [inline iOS/samples/.gitignore]: is this file necessary

File: iOS/samples/quickstart-swift/quickstart-swift/LaunchViewController.swift [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+        launchButton.widthAnchor.constraint(equalToConstant: 200).isActive = true
+        launchButton.heightAnchor.constraint(equalToConstant: 50).isActive = true
+        launchButton.centerXAnchor.constraint(equalTo: layoutGuide.centerXAnchor).isActive = true
+        launchButton.bottomAnchor.constraint(equalTo: layoutGuide.bottomAnchor, constant: -10).isActive = true
+        
+        // Create content and options.
+        sampleChunk = Chunk(content: bodyText.text!, lang: nil, mimeType: nil)
+        sampleContent = Content(title: titleText.text!, chunks: [sampleChunk])
+        sampleOptions = Options(uiLang: nil, timeout: nil, uiZIndex: nil)
+
```
Comment (rwallerms) [inline iOS/samples/quickstart-swift/quickstart-swift/LaunchViewController.swift]: nit: space

File: iOS/samples/quickstart-swift/quickstart-swift/LaunchViewController.swift [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+                }, onFailure: { error in
+                    self.launchButton.isEnabled = true
+                })
+            }
+            
+        }, onFailure: { error in
+                    print("an error occured: \(error)")
+            
+        })
+
```
Comment (rwallerms) [inline iOS/samples/quickstart-swift/quickstart-swift/LaunchViewController.swift]: nit: space

File: iOS/samples/quickstart-swift/quickstart-swiftTests/quickstart_swiftUnitTests.swift [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+            
+        }, onTimeout: { timeout in
+            XCTAssert(false)
+
+        }, onError: { error in
+            XCTAssert(false)
+            
+        })
+        
+        immersiveReaderViewController.src = "test://learningtools.onenote.com/learningtoolsapp/cognitive/reader"
```
Comment (rwallerms) [inline iOS/samples/quickstart-swift/quickstart-swiftTests/quickstart_swiftUnitTests.swift]: shoould define url as a constant

File: iOS/immersive-reader-sdk/.gitignore [hunk 5]
```diff
@@ -0,0 +1,79 @@
+
```
[the remark this answers is not in the document]
Comment (MeganRoach) [reply iOS/immersive-reader-sdk/.gitignore]: Yes, I believe so

File: iOS/samples/.gitignore (hunk 1 above)
Comment (MeganRoach) [reply iOS/samples/.gitignore]: Yes, I believe so

File: iOS/immersive-reader-sdk/immersive-reader-sdk.podspec [hunk 6]
```diff
@@ -0,0 +1,22 @@
+Pod::Spec.new do |spec|
```
[the remark this answers is not in the document]
Comment (MeganRoach) [reply iOS/immersive-reader-sdk/immersive-reader-sdk.podspec]: This is how Xcode does it's file naming by default

File: iOS/samples/quickstart-swift/quickstart-swift/LaunchViewController.swift (hunk 2 above)
Comment (MeganRoach) [reply iOS/samples/quickstart-swift/quickstart-swift/LaunchViewController.swift]: What do I need to change with the spacing?

File: iOS/immersive-reader-sdk/.gitignore (hunk 5 above)
[the remark this answers is not in the document]
Comment (metanMSFT) [reply iOS/immersive-reader-sdk/.gitignore]: I think you should combine this with the .gitignore in the root folder

File: iOS/immersive-reader-sdk/immersive-reader-sdk.podspec (hunk 6 above)
[the remark this answers is not in the document]
Comment (metanMSFT) [reply iOS/immersive-reader-sdk/immersive-reader-sdk.podspec]: I agree that it should be under `src`

File: iOS/immersive-reader-sdk/immersive-reader-sdk.podspec [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+  spec.name         = "immersive-reader-sdk"
+  spec.version      = "1.0.0"
+  spec.summary      = "Microsoft Cognitive Services - Immersive Reader iOS SDK"
+
+  spec.description  = <<-DESC
+                      Integrate Microsoft Cognitive Services Immersive Reader iOS SDK into your iOS App.
+                   DESC
+  spec.homepage     = "https://github.com/microsoft/immersive-reader-sdk/iOS"
+  spec.license      = "MIT"
+  spec.author       = { "Immersive Reader SDK" => "MeganRoach" }
```
Comment (metanMSFT) [inline iOS/immersive-reader-sdk/immersive-reader-sdk.podspec]: Not to discredit your work, but I think the author should be "Microsoft"

File: iOS/immersive-reader-sdk/immersive-reader-sdk.podspec [hunk 8]
```diff
@@ -0,0 +1,22 @@
...
+  spec.platform     = :ios, "9.0"
+
+  spec.source       = { :git => "https://github.com/microsoft/immersive-reader-sdk/immersive-reader-sdk/iOS.git", :branch => "master", :tag => "1.0.0" }
```
Comment (metanMSFT) [inline iOS/immersive-reader-sdk/immersive-reader-sdk.podspec]: This is not the right URL

File: iOS/immersive-reader-sdk/immersive-reader-sdk.podspec (hunk 6 above)
[the remark this answers is not in the document]
Comment (MeganRoach) [reply iOS/immersive-reader-sdk/immersive-reader-sdk.podspec]: After further discussion it has been decided that this will be left as is

File: iOS/immersive-reader-sdk/immersive-reader-sdk.podspec (hunk 7 above)
Comment (metanMSFT) [reply iOS/immersive-reader-sdk/immersive-reader-sdk.podspec]: irsdk@microsoft.com
